source/frontend/src/features/properties/list/columns.tsx

```diff
@@ -18,59 +23,108 @@ const NumberCell = ({ cell: { value } }: CellProps<IProperty, number>) => format
-    align: 'left',
-    minWidth: 100,
-    width: 150,
-  },
-  {
-    Header: 'Location',
-    accessor: p => p.address?.municipality,
-    align: 'left',
-    width: 50,
-    filter: {
-      component: TypeaheadField,
-      props: {
-        name: 'municipality',
-        placeholder: 'Filter by location',
-        className: 'location-search',
-        options: municipalities.map(mapLookupCode).map(x => x.label),
-        clearButton: true,
-        hideValidation: true,
+export function createTableColumns(
+  { municipalities }: Props,
+  onShowPropertyInfo: (note: IProperty, inNewTab: boolean) => void,
+): ColumnWithProps<IProperty>[] {
+  const columns: ColumnWithProps<IProperty>[] = [
+    {
+      Header: 'PID',
+      accessor: 'pid',
+      align: 'right',
+      width: 40,
+    },
+    {
+      Header: 'PIN',
+      accessor: 'pin',
+      align: 'right',
+      width: 40,
+    },
+    {
+      Header: 'Civic Address',
+      accessor: p => formatStreetAddress(p.address),
+      align: 'left',
+      minWidth: 100,
+      width: 150,
+    },
+    {
+      Header: 'Location',
+      accessor: p => p.address?.municipality,
+      align: 'left',
+      width: 50,
+      filter: {
+        component: TypeaheadField,
+        props: {
+          name: 'municipality',
+          placeholder: 'Filter by location',
+          className: 'location-search',
+          options: municipalities.map(mapLookupCode).map(x => x.label),
+          clearButton: true,
+          hideValidation: true,
+        },
       },
     },
-  },
-  {
-    Header: 'Lot Size (in\u00A0ha)',
-    accessor: 'landArea',
-    Cell: NumberCell,
-    align: 'right',
-    width: 20,
-    filter: {
-      component: Input,
-      props: {
-        field: 'maxLotSize',
-        name: 'maxLotSize',
-        placeholder: 'Filter by Lot Size',
-        className: 'filter-input-control',
-        type: 'number',
+    {
+      Header: 'Lot Size (in\u00A0ha)',
+      accessor: 'landArea',
+      Cell: NumberCell,
+      align: 'right',
+      width: 20,
+      filter: {
+        component: Input,
+        props: {
+          field: 'maxLotSize',
+          name: 'maxLotSize',
+          placeholder: 'Filter by Lot Size',
+          className: 'filter-input-control',
+          type: 'number',
+        },
       },
     },
-  },
-];
+    {
+      Header: '',
+      accessor: 'controls' as any, // this column is not part of the data model
+      align: 'right',
+      sortable: false,
+      width: 20,
+      Cell: (cellProps: CellProps<IProperty, number>) => {
+        const { hasClaim } = useKeycloakWrapper();
+
+        return (
+          <StyledDiv>
+            {hasClaim(Claims.PROPERTY_VIEW) && (
+              <StyledIconButton
+                data-testid="view-prop-tab"
+                title="View Property Info"
+                variant="light"
+                onClick={() => {
+                  onShowPropertyInfo(cellProps.row.original, false);
+                }}
```

Comment:
See my previous comment re: using router links instead of callback functions here...